2.0/s2i/bin/run

```diff
@@ -10,6 +10,16 @@ set -e
 export RACK_ENV=${RACK_ENV:-"production"}
 
 if is_puma_installed; then
+  export_str=$(cgroup-limits.py)
+  [[ -n "$export_str" ]] && export $export_str
+
+  if [[ -z "${MEMORY_LIMIT_IN_BYTES:-}" ]]; then
+    echo "Warning: Can't detect memory limit"
+  fi
+  if [[ -z "${NUMBER_OF_CORES:-}" ]]; then
+    echo "Warning: Can't detect number of cores"
```

Comment:
If the script errors out, it will exit. But yeah, now that we print the maximum value we can assume it'll never be empty. The script will ignore missing cgroup files, that's why the warning is there. If we want to assume they will always be there, then we can remove the warnings and make the cgroup script not ignore them. Do we want to do that, though?